### PR TITLE
Get the current logged in user info

### PR DESCRIPTION
**Pull request by JamesFerretti**

### CODE REVIEW

File: source/includes/reference/_users.md [hunk 1]
```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
...
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+{
+  "id": 294205,
+  "name": "Oliver Wilson",
+  "email": "oliver@gmail.com",
```
Comment (kdowbecki) [inline source/includes/reference/_users.md]: I'd advise against using possibly existing emails, change to something like `oliver.wilson@example.com`

File: source/includes/reference/_users.md [hunk 2]
```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
...
+curl -X GET https://api.sandbox.transferwise.tech/v1/me \
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+{
+  "id": 294205,
+  "name": "Oliver Wilson",
```
Comment (kdowbecki) [inline source/includes/reference/_users.md]: Is this an actual user? If so is it ok to show his data in docs?

File: source/includes/reference/_users.md [hunk 3]
```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
...
+email                   | User email                     | Text
+active                  | If user is active or not       | Boolean
+details.firstName       | User first name                | Text
+details.lastName        | User lastname                  | Text
+details.phoneNumber     | Phone number                   | Text
+details.dateOfBirth     | Date of birth                  | YYYY-MM-DD
+details.occupation      | Person occupation              | Text
+details.avatar          | Link to person avatar image    | Text
+details.primaryAddress  | Address object id              | Integer
+details.address.country  | Address country code in 2 digits. "US" for example            | Text
```
Comment (kdowbecki) [inline source/includes/reference/_users.md]: Should be `countryCode` as per example response above. Also are we sure this is 2 characters? In many places we use 3 character ISO code.

File: source/includes/reference/_users.md [hunk 4]
```diff
@@ -62,7 +62,67 @@ details.address.city        | Address city name          | Text
...
+    "address": {
+      "city": "Tallinn",
+      "countryCode": "EE",
+      "postCode": "12112",
+      "state": "",
+      "firstLine": "Salu tee 134"
+    },
+    "dateOfBirth": "1977-07-01",
+    "avatar": "https://lh6.googleusercontent.com/-XzeFZ2PJE1A/AAAAAAAAAAI/AAAAAAAAAAA/RvuvhXFsqs0/photo.jpg",
+    "primaryAddress": 236532
```
Comment (kdowbecki) [inline source/includes/reference/_users.md]: Makes no sense in the response, what would we use the `primaryAddress` field for?

File: source/includes/reference/_users.md (hunk 3 above)
Comment (JamesFerretti) [reply source/includes/reference/_users.md]: Thanks, yes it is 2 char

File: source/includes/reference/_users.md (hunk 4 above)
Comment (JamesFerretti) [reply source/includes/reference/_users.md]: You use it to look up in the address endpoints

File: source/includes/reference/_users.md (hunk 2 above)
Comment (JamesFerretti) [reply source/includes/reference/_users.md]: Will modify

**Pull request merged**